src/css/components/_modal-dialog.scss

```diff
@@ -18,6 +18,8 @@
 .vjs-modal-dialog .vjs-modal-dialog-content {
   @extend %fill-parent;
 
+  @include user-select(auto);
```

Comment:
A thought: what about if we just disabled user selection on the progress bar if that's what we're trying to guard against?